graalwasm/graalwasm-embed-c-code-guide/src/main/java/com/example/App.java

```diff
@@ -17,20 +17,25 @@ public class App {
     public static void main(String[] args) throws IOException {
         // Find the WebAssembly module resource
         URL wasmFile = App.class.getResource("floyd.wasm");
+        Source source = Source.newBuilder("wasm", wasmFile).name("example").build();
```

Comment:
We can drop `.name("example")` as the `Source` name is no longer used when reading the exports.